web/secondary_toolbar.js

```diff
@@ -79,27 +80,46 @@ var SecondaryToolbar = {
         element.addEventListener('click', elements[item].handler.bind(this));
       }
     }
+
+    // Tracking hand tool menu item changes.
```

Comment:
With these changes, this line can be removed https://github.com/yurydelendik/pdf.js/blob/secondary/web/secondary_toolbar.js#L61.